pynetsnmp/netsnmp.py

```diff
@@ -472,7 +472,7 @@ def awaitTraps(self, peername, fileno = -1, pre_parse_callback=None, debug=False
             lib.netsnmp_udp6_ctor()
         else:
             log.debug("Cannot find constructor function for UDP/IPv6 transport domain object.")
-        lib.init_snmpv3(None)
+        lib.init_snmp("zenoss_app")
```

Comment:
Why is `init_snmp` rather than `init_snmpv3` the correct function to call?  Isn't this about the SNMPv3 protocol?